user_sync/config.py

```diff
@@ -334,7 +335,7 @@ def get_rule_options(self):
         if (extensions_config is not None):
             for extension_config in extensions_config.iter_dict_configs():
                 context = extension_config.get_string('context')
-
+                
```

Comment:
Was there a reason for this change?  Also, not a good idea to have spurious whitespace at the front of a line.